wrangler.toml

```
@@ -1,6 +1,6 @@
-name = "stdrs"
-type = "webpack"
-webpack_config = "./webpack.config.js"
+name = "std-rs"
```

Accidentally changed this, revert back to
```suggestion
name = "stdrs"
```